src/java/org/apache/cassandra/cql3/ColumnReference.java

```diff
@@ -104,7 +106,7 @@ public ValueReference toValueReference(QueryOptions options)
 
     public ColumnIdentifier getFullyQualifiedName()
     {
-        String fullName = selectName + '.' + column.name.toString() + (cellPath == null ? "" : '[' + cellPath.toString() + ']');
+        String fullName = selectName.name() + '.' + column.name.toString() + (cellPath == null ? "" : '[' + cellPath.toString() + ']');
```

Comment:
using `.name()` here as we know its a `user` type; we don't use this when dealing with `auto-gen` reads